atat_provisioning_wizard_api.yaml

```
@@ -439,6 +439,13 @@ components:
             name:
               type: string
               description: name of file when originally uploaded
+            status:
+              type: string
+              description: virus scanning status of uploaded file
+              enum:
+                - "pending"
+                - "clean"
+                - "dirty"
```

```suggestion
            status:
              type: string
              description: status of uploaded file scans and review
              enum:
                - "pending"
                - "accepted"
                - "rejected"
```

should we make this a little more generic and avoid `clean` or `dirty`? gives us room to do more than just virus scanning

---

we could have a "rejection reason" or something like that too if we needed